components/schema-migrator/migrations/director/202001311508_migrate_tenants.up.sql

```diff
@@ -0,0 +1,29 @@
+
+insert into business_tenant_mappings(id)
+select tenant_id from applications a2 union select tenant_id from runtimes r2;
+update business_tenant_mappings set external_tenant = id;
+update business_tenant_mappings set external_name = 'Tenant';
```

Comment:
Add Where condition, because you will change all `extenal_name` and `external_tenant`.